cookbook/migrations/0121_auto_20210518_1638.py

```diff
@@ -18,6 +18,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='userpreference',
             name='use_fractions',
-            field=models.BooleanField(default=False),
+            field=models.BooleanField(default=True),
```

Comment:
as only the Americans use fractions as far as i know and decimals are far superior in my opinion i would like to leave the default as decimals.

Reply:
for some reason that keep leaking in - every time I run a migration it adds it in.  I usually take it out but must have forgotten one.